schema/show.js

```diff
@@ -315,7 +315,7 @@ const Show = {
       .then(show => {
         if (!show.displayable && !show.is_reference) return new Error('Show Not Found');
         return show;
-      });
+      }).catch(() => null);
```

Comment:
Yea good point, I can amend this so it only catches 404s there.

Something I was thinking, but I don't know about GraphQL to know if it's possible/doable - it's only one specific endpoint that might return these badly cached items.

So, only when resolving a show from one of those endpoints do we want to catch.

Something like:

```
show(id: "invalid") {
  id
}
```

The above would 404 since the show isn't found.

However,

```
artist(id: "andy-warhol") {
  exhibition_highlights {
    name
    ...more attributes
  }
}
```

The above we'd want to silently fail on some of those deeper attributes. Is it possible to know in a resolver where in the query it's being invoked? I looked around at AST but didn't find anything so just rolled with this.

"when the only tool you have is a hammer"